### DISCUSSION TITLE
I lost my google authenticator

### DISCUSSION THREAD

**Discussion by Saidoubri** [Authenticator-Extension/Authenticator#871] [General]
Hello support how i can recovery my google authenticator pls

**Comment by Kashif9601** [comment]
Dear Support team,

need your urgent help.

Hi google team,

please be request i had reset my device mobile huwaie y9.
Google authenticate was insides i did not take backup due to issue in mobile.

now i am trying to login but i do not have key. so please help me out to get rid off from this issues.

**Comment by Sneezry** [comment]
This project has no relationship with Google's mobile app
